deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```
@@ -630,7 +631,7 @@ service:
     ## - added transform/remove_attributes processor
     logs/otlp/kubelet:
       exporters:
-        - sumologic/systemd
+{{ include "logs.otelcol.systemd.exporters" . | indent 8 }}
```

Did you want `kubelet` here, as you did above in line 547?
```suggestion
{{ include "logs.otelcol.kubelet.exporters" . | indent 8 }}
```

---

Yes, good catch. They're the same right now, but could be different in the future.